apps/web/src/app/(signing)/sign/[token]/form.tsx

```diff
@@ -74,7 +75,7 @@ export const SigningForm = ({ document, recipient, fields }: SigningFormProps) =
       timestamp: new Date().toISOString(),
     });
 
-    router.push(`/sign/${recipient.token}/complete`);
+    redirectUrl ? router.push(redirectUrl) : router.push(`/sign/${recipient.token}/complete`);
```

Comment:
Will that work? I assumed it might need a dirty `window.location.href = redirectUrl` 🤔